p2p/test/transport/transport_test.go

```diff
@@ -368,10 +368,15 @@ func TestMoreStreamsThanOurLimits(t *testing.T) {
 			}))
 
 			var handledStreams atomic.Int32
+			var sawFirstErr atomic.Bool
```

Comment:
This `atomic.Bool` is only ever loaded, but never set. Or am I missing something?